src/scss/utilities/_focus-ring.scss

```diff
@@ -1,10 +1,5 @@
 /// FOCUS RING
```

Comment:
btw it looks like rounded outlines finally landed in Canary as of today I think

Reply:
I think when it hits main channel, we get rid of/update this util then